enos/enos-variables.hcl

```
@@ -124,3 +124,9 @@ variable "local_build_target" {
   type        = string
   default     = "build-ui build"
 }
+
+variable "e2e_debug" {
```

The name of this variable doesn't clearly communicate the effect IMO, what do you think about 
```suggestion
variable "e2e_debug_no_run" {
```
Or something?